packages/react-components/src/Concepts/Slots.stories.mdx

```diff
@@ -61,8 +61,8 @@ It also has an `icon` slot that only renders when there is neither an image nor
 
 A property of a component can be identified as a slot by its type.
 
-- IntrinsicSlotProps: The props for a slot that supports one or more HTML element types, such as 'div'.
-- ComponentSlotProps: The props for a slot that additionally supports a JSX component or component props object.
+- Slot: The props for a slot that supports one or more HTML element types like 'div', or component type like Button.
+- SlotNoChildren: Same as Slot, but does not allow shorthand for children.
 - ShorthandRenderFunction: A render function for a slot.
```

Comment:
PR description mentions `WithSlotRenderFunction`, should be this `SlotRenderFunction`?

Reply:
Cleaned up this documentation to only mention the `Slot` type, since that's now the One True Type for slots.